ui/tests/helpers/commands.js

```diff
@@ -31,30 +36,47 @@ import { create } from 'ember-cli-page-object';
  * }
  */
 
-const cc = create(consoleClass);
-
 /**
  * runCmd is used to run commands and throw an error if the output includes "Error"
  * @param {string || string[]} commands array of commands that should run
  * @param {boolean} throwErrors
  * @returns the last log output. Throws an error if it includes an error
  */
-export async function runCmd(commands, throwErrors = true) {
+export const runCmd = async (commands, throwErrors = true) => {
   if (!commands) {
     throw new Error('runCmd requires commands array passed in');
   }
   if (!Array.isArray(commands)) {
     commands = [commands];
   }
-  await cc.toggle();
-  await cc.runCommands(commands, false);
-  const lastOutput = cc.lastLogOutput;
-  await cc.toggle();
+  await click(REPL.toggle);
+  await enterCommands(commands);
+  const lastOutput = await lastLogOutput();
+  await click(REPL.toggle);
   if (throwErrors && lastOutput.includes('Error')) {
     throw new Error(`Error occurred while running commands: "${commands.join('; ')}" - ${lastOutput}`);
   }
   return lastOutput;
-}
+};
+
+export const enterCommands = async (commands) => {
+  const toExecute = Array.isArray(commands) ? commands : [commands];
+  for (const command of toExecute) {
+    await fillIn(REPL.consoleInput, command);
+    await triggerKeyEvent(REPL.consoleInput, 'keyup', 'Enter');
+  }
+};
+
```

Comment:
this replaces https://github.com/hashicorp/vault/blob/530b26608eee62a5820344cd8b1f5475f58e2017/ui/tests/pages/components/console/ui-panel.js#L21-L29